src/main/java/com/example/PicturesToNumber/nn/Initialize.java

```diff
@@ -0,0 +1,30 @@
+package com.example.PicturesToNumber.nn;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+@Component
+@PropertySource(name = "appProperties", value = "application.properties")
+public class Initialize {
+
+    public NeuralNetwork network;
+
+    @Value("#{'${targetWidth}'}")
```

Comment:
Why so difficult? @Value("#{'${targetWidth}'}") Please check this example https://docs.spring.io/spring-boot/docs/2.1.13.RELEASE/reference/html/boot-features-external-config.html